src/foam/nanos/dig/DIG.js

```diff
@@ -148,7 +150,7 @@ foam.CLASS({
       label: 'Send Request',
       code: function() {
         var req = this.HTTPRequest.create({
-          url: window.location.origin + this.digURL,
+          url: this.appConfig.URL.value + this.digURL.substring(1),
```

Comment:
This is incorrect. You need to get the user, get the group and get the appConfig from the group. 
@ZacZhang and @sirenChen can provide guidance.

Reply:
AppConfig appears to be setup correctly on the client.